analytics/views.py

```diff
@@ -1,3 +1,30 @@
 # from django.shortcuts import render
+from django.utils.decorators import decorator_from_middleware
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from . import middleware, models, serializers
 
 # Create your views here.
+
+
+@api_view(['GET'])
+def home(_request):
+    """Render the most basic home page ever designed."""
+    return Response('Analytics app seems to be running.')
+
+
+@api_view(['GET', 'POST'])
+@decorator_from_middleware(middleware.AnalyticsTokenAuthMiddleware)
+def event_list(request):
+    """List all events or create a new event."""
+    if request.method == 'GET':
+        events = models.Event.objects.all()
+        serializer = serializers.EventSerializer(events, many=True)
+        return Response(serializer.data)
+    if request.method == 'POST':
+        serializer = serializers.EventSerializer(data=request.data)
```

Comment:
- serializer przy tworzeniu przyjmuje created_at (sql i tak wstawi swoją date, ale można było to rozdzielić na EventGetSerializer oraz EventCreateSerializer)

Reply:
Można, celowo nie rozdzielałem. Korzyści z takich decyzji byłyby potencjalne, nie dotyczyłoby wprost realizacji punktów wprost zapisanych w zadaniu. Działanie jest poprawne, a domyślne rozwiązania są moim zdaniem czytelniejsze (szczególnie przy generic view, gdzie get_serializer w tym przypadku zawsze wystarczy).